src/main/java/com/starbucks/model/Order.java

```diff
@@ -1,81 +1,50 @@
 package com.starbucks.model;
 
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import java.util.UUID;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
 
+import javax.persistence.*;
+import java.util.UUID;
 
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@Builder
+@Table(name = "Order", schema = "starbucks", catalog = "postgres")
```

Comment:
Do you really need catalog and schema? You can put schema in your application properties file.

Reply:
Yes, I have a schema in properties.